.rubocop_todo.yml

```diff
@@ -1,14 +1,11 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config --exclude-limit 500`
-# on 2019-07-20 01:16:04 -0500 using RuboCop version 0.72.0.
+# on 2019-07-23 14:13:22 -0400 using RuboCop version 0.65.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 1
-# Configuration parameters: MinNameLength, AllowNamesEndingInNumbers, AllowedNames, ForbiddenNames.
-# AllowedNames: io, id, to, by, on, in, at, ip, db
-Naming/UncommunicativeMethodParamName:
+Style/FrozenStringLiteralComment:
```

Comment:
One last change.
Not sure why this changed, but it still needs to be `Naming/UncommunicativeMethodParamName:` to catch the known List offense of the "lg" prop.